internal/cmd/restore.go

```diff
@@ -32,27 +32,39 @@ var restoreCmd = &cobra.Command{
 	RunE:  restoreCmdFunc,
 }
 
-func restoreCmdFunc(cmd *cobra.Command, args []string) error {
-	filename := args[0]
+func openRestoreFile(filename string) (*os.File, int64, error) {
+	if filename == "-" {
```

Comment:
Instead of using `-` here, should `stdin` just be the default if no filename is given?

Reply:
Done.